### PR TITLE
remove dependency on hard coded table names

### PR DESCRIPTION
**Pull request by snopoke**
pre work for changing table name length

### CODE REVIEW

**Review by emord — APPROVED** [review]

File: corehq/apps/userreports/tests/test_utils.py [hunk 1]
```diff
@@ -43,7 +43,7 @@ def test_table_trickery(self):
 
     def test_long_table_name(self):
         name = get_table_name('this_is_a_long_domain', 'and_a_long_table_name')
-        name_expected = 'config_report_this_is_a_long_domain_and_a_long_table_n_6ac28759'
+        name_expected = '{}this_is_a_long_domain_and_a_long_table_n_6ac28759'.format(UCR_TABLE_PREFIX)
```
Comment (calellowitz) [inline corehq/apps/userreports/tests/test_utils.py]: This hash is going to change (the hash when the table name gets truncated includes the prefix)

**Review by calellowitz — APPROVED** [review]
Looks good once tests pass, although I suspect you will have to fix the one comment

File: corehq/apps/userreports/tests/test_utils.py (hunk 1 above)
Comment (snopoke) [reply corehq/apps/userreports/tests/test_utils.py]: yes

**Comment by snopoke** [comment]
seems I missed one - running tests locally before pushing changes

**Review by emord — APPROVED** [review]

**Pull request merged**